index.html

```diff
@@ -2,9 +2,174 @@
 <html lang="en">
 <head>
     <meta charset="UTF-8">
-    <title>Title</title>
+    <title>Een blok positioneren</title>
+    <link rel="stylesheet" href="./styles.css">
 </head>
 <body>
+<header class="header__outer-container">
+    <div class="header__inner-container">
+        <nav class="header__navbar">
+            <ul class="header__navbar-list">
+                <li class="header__navbar-item"><a href="./pages/not found">case studies</a></li>
+                <li class="header__navbar-item"><a href="">services</a></li>
+                <li class="header__navbar-item"><a href="">contact us</a></li>
+            </ul>
+        </nav>
+        <div class="header__hero-section">
+            <h1 class="header__title">Your website is an investment in the<br> succes of your business.</h1>
+            <p class="header__payoff">We are here tot help you succeed. Let's work</p>
+            <button class="button--active">let's work together</button>
+        </div>
+        <div class="header__squiggle__line_strip"><object><embed src="./assets/squiggle_line_stripe.svg"></object></div></div>
+
+</header>
+
+<main>
+    <!-- intro -->
+    <section class="intro__outer-container">
+        <div class="intro__inner-container">
+
+            <!--intro text -->
+            <section class="intro__section">
+                <h2 class="intro--text">Rekupe is a digital agency in Los Angeles.<br>
+                    We us strategic UX, SEO optimization and beautiful visual designs to help your business grow.<br> We
+                    see every project as a fresh opportunity, pushing <br> ourselves to creat new digital experiences.
+                </h2>
+
+                <!--Intro button-->
+                <div class="intro__section-question">
+                    <h5>
+                        <p>Have a question, </p>
+                        <input type="submit" value="Click here">
+                        <p> to contact us! </p></h5>
+                </div>
+        </div>
+    </section>
+</main>
+<main>
+    <!--projects -->
+    <!--section 1-->
+     <section class="projects__outer-container">
+        <article class="projects__inner-container">
+            <div class="projects__article-image-wrapper-v1">
+                <div class="projects__card">
+                    <h2 class="projects__title">rolling stone</h2>
+                    <div class="projects__squiggle_line_strip">
+                            <embed src="./assets/squiggle_line.svg">
```

Comment:
Hier plaats je de img in een `<div>` en een paar regels verder in een `<object>`. Gebruik hierbij 1 standaard. Tevens zou ik je aanraden een image eerder in een `<span>` tag te omwikkelen. Een span is een inline element en een div is een block element. Een span element heeft dan meer styling vrijheid.